src/frontend/typer/manuell-journalføring.ts

```diff
@@ -65,3 +65,55 @@ export interface INavnOgIdent {
     navn: string;
     id: string;
 }
+
+export enum JournalpostTittel {
+    SØKNAD_OM_BARNETRYGD_ORDINÆR = 'Søknad om barnetrygd ordinær',
+    SØKNAD_OM_UTVIDET_BARNETRYGD = 'Søknad om utvidet barnetrygd',
+    ETTERSENDELSE_TIL_SØKNAD_OM_BARNETRYGD_ORDINÆR = 'Ettersendelse til søknad om barnetrygd ordinær',
+    ETTERSENDELSE_TIL_SØKNAD_OM_UTVIDET_BARNETRYGD = 'Ettersendelse til søknad om utvidet barnetrygd',
+    TILLEGGSSKJEMA_VED_KRAV_OM_UTBETALING = 'Tilleggsskjema ved krav om utbetaling av barnetrygd og/eller kontantstøtte på grunnlag av regler om eksport etter EØS-avtalen',
+}
+
+export const BrevkodeMap = new Map([
+    [JournalpostTittel.SØKNAD_OM_BARNETRYGD_ORDINÆR, 'NAV 33-00.07'],
+    [JournalpostTittel.SØKNAD_OM_UTVIDET_BARNETRYGD, 'NAV 33-00.09'],
+    [JournalpostTittel.ETTERSENDELSE_TIL_SØKNAD_OM_BARNETRYGD_ORDINÆR, 'NAV 33-00.07'],
+    [JournalpostTittel.ETTERSENDELSE_TIL_SØKNAD_OM_UTVIDET_BARNETRYGD, 'NAV 33-00.09'],
+    [JournalpostTittel.TILLEGGSSKJEMA_VED_KRAV_OM_UTBETALING, 'NAV 34-00.15'],
+]);
+
+export enum DokumentTittel {
```

Comment:
I would also split this actually. Maybe some others on the team have other opions?

Reply:
We think the current implementation also works well and is simpler, but if there is explicit disadvantage, we would like to change as well.

Reply:
It works indeed, but we have den similar bugs from the same setup ☺️ We can discuss more in detail om call if you'd like 👍